src/main/groovy/nebula/plugin/publishing/maven/ResolvedMavenPlugin.groovy

```diff
@@ -51,6 +51,10 @@ class ResolvedMavenPlugin implements Plugin<Project> {
                         [scope, resolutionMap]
                     }
 
+                    // Another plugin may have already patched the POM to support 'provided' scope.
+                    // For the purpose of version resolution, 'provided' scope is equavalent to 'runtime'.
+                    scopeResolutionMap['provided'] = scopeResolutionMap['runtime']
```

Comment:
Is this going to be ordering dependent? If another plugin configures provided in this Map will the ordering the plugins are applied matter?